### PR TITLE
"Final Review" layout

### PR DESCRIPTION
**Pull request by kyleknighted**
Integrates some general layout and styles for Review step

127 0 0 1_8080_aws_cloudwatch (2)

### CODE REVIEW

File: src/web/aws-cloudwatch/CloudWatch.jsx [hunk 1]
```diff
@@ -3,11 +3,13 @@ import PropTypes from 'prop-types';
 import { Col, Row } from 'react-bootstrap';
 
 import Wizard from 'components/common/Wizard';
+import PageHeader from 'components/common/PageHeader';
 
 import StepAuthorize from './StepAuthorize';
 import StepKinesis from './StepKinesis';
 import StepHealthCheck from './StepHealthCheck';
 import StepReview from './StepReview';
+import { DEFAULT_SETTINGS } from './util';
```
Comment (dennisoelkers) [inline src/web/aws-cloudwatch/CloudWatch.jsx]: I think we should rename this to `defaults.js` or `default_settings.js` to make its intention clearer.

File: src/web/aws-cloudwatch/util.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  awsCloudWatchGlobalInput: '',
+  awsCloudWatchAssumeARN: '',
+  awsCloudWatchBatchSize: '10000',
+  awsCloudWatchThrottleEnabled: '',
+  awsCloudWatchThrottleWait: '1000',
+};
+
+export { DEFAULT_SETTINGS };
+
+export default {};
```
Comment (dennisoelkers) [inline src/web/aws-cloudwatch/util.js]: I think we can safely make `DEFAULT_SETTINGS` the default export and use the file solely for it.

File: src/web/aws-cloudwatch/StepReview.jsx [hunk 3]
```diff
@@ -1,20 +1,78 @@
...
+                <span>{defaultOutput('awsCloudWatchBatchSize')}</span>
+              </li>
+              <li>
+                <strong>Throttled Wait (ms)</strong>
+                <span>{defaultOutput('awsCloudWatchThrottleWait', values.awsCloudWatchThrottleEnabled)}</span>
+              </li>
+            </ReviewItems>
+
+            <Subheader>Formatting <FormatIcon success><i className="fa fa-smile-o" /></FormatIcon></Subheader>
+            <p>Parsed as LogFlow, if you need a different type you&apos;ll need to setup a <Link to={Routes.SYSTEM.PIPELINES.RULES}>Pipeline Rule</Link>.</p>
```
Comment (danotorrey) [inline src/web/aws-cloudwatch/StepReview.jsx]:
```suggestion
            <p>Parsed as FlowLog, if you need a different type you&apos;ll need to setup a <Link to={Routes.SYSTEM.PIPELINES.RULES}>Pipeline Rule</Link>.</p>
```

File: src/web/aws-cloudwatch/CloudWatch.jsx [hunk 4]
```diff
@@ -16,6 +18,24 @@ export default class AWSCloudWatch extends Component {
     }).isRequired,
   }
 
+  logOutput = { // Demo Data until API is wired
```
Comment (danotorrey) [inline src/web/aws-cloudwatch/CloudWatch.jsx]: The change to include these fields is the Health Check response (#114) has now been merged into the `aws` branch.

**Pull request merged**